com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/GameProjectBuilder.java

```diff
@@ -412,6 +412,11 @@ private static void buildResourceGraph(Project project, IResource resource, Reso
             }
         }
 
+        // Editor debugger scripts
```

Comment:
This will make sure that all required files are included:

* builtins/scripts/debugger.lua
* builtins/scripts/edn.lua
* builtins/scripts/mobdebug.lua
* builtins/scripts/socket.lua